include/knowhere/config.h

```diff
@@ -641,8 +641,8 @@ class BaseConfig : public Config {
     CFG_BOOL enable_mmap_pop;
     CFG_BOOL for_tuning;
     CFG_BOOL shuffle_build;
-    CFG_BYTES trace_id;
-    CFG_BYTES span_id;
+    CFG_STRING trace_id;
```

Comment:
Why do we need to include trace in the parameter set? Logically, the index doesn't consume this parameter; it's just used for some chain tracing. Can we consider constructing the TraceContext externally and passing it through the interface, extending the parameters on the interface to achieve this?

Reply:
Adding traceID and spanID into config is the most economical way to achieve this.
If change the interfaces for Build()/Search()/RangeSearch(), then pyknowhere API need update (if I remember correctly, pyknowhere API does not support default input parameter), and Milvus code also need change.